### PR TITLE
Code cleanup and Now update

### PR DESCRIPTION
**Pull request by raulg**
Linter fix for consistent code that should be easier to follow and understand.
Updated included dev dependency on Now to 15.3.0 for `now dev` routing fixes
Changed default development tool to `now dev` from `nodemon server.js`
Updated Read-me to reflect these changes and a link to the new Next.js article in docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,13 +29,13 @@ As of version 15.0.0 of `now` you can use the `now dev` command to run the proje
...
-> $ npm run now-dev
+> $ npm run dev
 ```
 
 ### Alternative local development mode
-If you find required rebuild process with `now dev` slows down your development workflow, you can run the included Express server file which will handle the same custom routing needs. Keep in mind that the routes defined here are different from the ones defined in `now.json`, always check your routes with `now dev` before deploying live to production. Development server will be running in `http://localhost:3000/` 
+If your workflow favors working with Node, you can run the included Express server file which will handle the same custom routing needs. Keep in mind that the routes defined here are different from the ones defined in `now.json`, make sure to replicate and check your routing before deploying to production on Now. Development server will be running in `http://localhost:3000/` 
 ```
-> $ npm run dev
+> $ npm run nodev
```
Comment (levimykel) [inline README.md]: Please update to node-dev

File: package.json [hunk 2]
```diff
@@ -8,26 +8,26 @@
...
-    "now": "^15.0.1"
+    "express": "^4.17.0",
+    "nodemon": "^1.19.0",
+    "now": "^15.3.0"
   },
   "scripts": {
-    "dev": "nodemon server.js",
-    "now-dev": "now dev",
+    "dev": "now dev",
+    "nodev": "nodemon server.js",
```
Comment (levimykel) [inline package.json]: Also change this to node-dev

**Review by levimykel — CHANGES_REQUESTED** [review]
Everything looks good. Just change the `nodev` to `node-dev`, please.

**Comment by raulg** [comment]
Changes executed, should be safe to merge

**Pull request merged**